tmk_core/common/eeconfig.c

```diff
@@ -187,3 +187,7 @@ uint32_t eeconfig_read_haptic(void)      { return eeprom_read_dword(EECONFIG_HAP
  * FIXME: needs doc
  */
 void eeconfig_update_haptic(uint32_t val) { eeprom_update_dword(EECONFIG_HAPTIC, val); }
+
+bool eeconfig_read_handedness(void)      { return !!eeprom_read_byte(EECONFIG_HANDEDNESS); }
```

Comment:
Should this whole change be gated by `#ifdef SPLIT_KEYBOARD`?

Reply:
I would say no.  Since it's possible to convert a split board to a single non-split board, that may not be for the best.

Reply:
Thats a good point. I shall leave it as it is for now then.